vars/suse-15.yml

```diff
@@ -15,4 +15,4 @@
 # limitations under the License.
 
 _zypper:
-  - { state: "latest", name: "MozillaFirefox" }
+  - { state: "latest", name: "darktable" }
```

Comment:
Please add OBS repo for suse as mentioned in https://www.darktable.org/install/#3rdparty